src/main/java/com/yugabyte/sample/apps/AppBase.java

```diff
@@ -162,21 +175,62 @@ protected String getKeyspace() {
     return keyspace;
   }
 
+  /**
+   * Private function that returns an SSL Handler when provided with a cert file.
+   * Used for creating a secure connection for the app.
+   */
+  private SSLOptions createSSLHandler(String certfile) {
```

Comment:
Is there some sample code to follow on how this is supposed to be setup?

Reply:
https://docs.datastax.com/en/developer/java-driver/3.1/manual/ssl/

Reply:
https://netty.io/4.0/api/io/netty/handler/ssl/SslHandler.html